src/components/NotificationPanel/NotificationBox.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "./NotificationBox.css";
+import PropTypes from "prop-types";
+
+const NotificationBox = ({ text, color, time, image }) => (
+  <div className="NotificationBox">
+    <div className="color-type" style={{ backgroundColor: color }} />
+    <img className="image" src={image} alt="Notification" />
+    <p>
+      {text}
```

Comment:
Text formatting still needs some work. It's hard to show, but this formatting is off 
![image](https://user-images.githubusercontent.com/12878275/58134937-5e348580-7bdd-11e9-8c78-2f3e2cffb8f7.png)